test/spec.sh

```diff
@@ -986,6 +986,11 @@ ble-idioms() {
   run-file ble-idioms "$@"
 }
 
+ble-sparse() {
+  sh-spec spec/ble-sparse.test.sh \
```

Comment:
this should use `run-file ble-sparse "$@"` 

and then the top of the file should have

`## compare_shells: bash mksh`

That is the new style  -- and hopefully I can get rid of all these tiny functions at some point, once they are all consistent

Reply:
OK. I thought I just copied the configuration for `ble-idioms`, but `ble-idioms` seems to have been updated just yesterday.